models/indexModel.js

```diff
@@ -0,0 +1,20 @@
+/* eslint-disable no-console */
+const CSVToJSON = require('csvtojson')
+
+
+// index model here is reading a file  and returning response customers listed in the file
+// instead we should be reading from the database if it has info 
+// also display a mesg if it does not have info
+const indexModel = (req, res) => {
+  CSVToJSON()
```

Comment:
This code already exists in **dataController**, another cool term we use in programming is DRY, which stands for Do Not Repeat Yourself, where duplication of the code is bad and it is something we want to avoid. This a model file, where here should leave only code related with entities for example the create the model for the Customer.